Unix/provmgr/provmgr.c

```diff
@@ -165,16 +165,22 @@ static Library* MI_CALL _OpenLibraryInternal(
     {
         TChar path[PAL_MAX_PATH_SIZE];
         path[0] = '\0';
-        Shlib_Format(path, self->providerDir, proventry->libraryName);
+        Shlib_Format(path, self->providerDir,
+                     proventry->interpreter ? "ScriptProvider" :
```

Comment:
Wondering if this should be omiscriptprovider in case we have another provider that matches this.